src/analysis/parsetree_utils.mli

```diff
@@ -6,3 +6,5 @@ open Parsetree
 type nonrec constant_desc = constant_desc
 
 val constant_desc : constant -> constant_desc
+
+val filter_expr_attr : expression -> expression
```

Comment:
Can you add a small docstring that explain the behaviour of the function (it is less obvious than `constant_desc` :)

Reply:
Does the following comment do the job?
> Filter parsetree attributes that starts with ["merlin."] in given expression.

Reply:
I think that we can name it: `expr_remove_merlin_attributes` so?